content/en/docs/contribute/style/style-guide.md

```diff
@@ -58,9 +55,9 @@ leads to an awkward construction.
 {{< table caption = "Do and Don't - API objects" >}}
 Do | Don't
 :--| :-----
-The Pod has two containers. | The pod has two containers.
-The Deployment is responsible for ... | The Deployment object is responsible for ...
-A PodList is a list of Pods. | A Pod List is a list of pods.
+The pod has two containers. | The Pod has two containers.
+The HorizontalPodAutoscaler is responsible for ... | The HorizontalPodAutoscaler object is responsible for ...
+A PodList is a list of pods. | A Pod List is a list of pods.
```

Comment:
In the API, a PodList _is_ a list of Pods.

Reply:
I wasn't saying that's not the case. I'm just saying that the example was confusing, so trying to use one that is more clear to  show our meaning for when to capitalize vs not.

Reply:
Hello @cmastr .
I see that there is at least one remaining question to address _before_ these changes can merge.
Not to belabor the point but perhaps line 60 is not a fair example and should be removed?

`A PodList is a list of pods. | A Pod List is a list of pods.`

For example (of course the field names would use code style not double quotes):
`A ServiceAccount has twenty-nine fields such as "kind", "version", ...` versus `A Service account is used throughout your Kubernetes documentation` OR
`This page teaches you how to set the "kind" field in a Pod spec` versus `This page teaches you about a Pod spec`

I have not read your Kubernetes documentation, but am guessing that you are looking for some standardization of the Kubernetes terms so that you can more easily refer to this content.

Reply:
I agree that this example is not good. I was trying not to change so much other than the capitalization which was the intent of this PR. But the examples do definitely need some work. I'm good with removing line 60. Replacing it with something else that is more explanatory of what we're trying to get across. Will update.

Reply:
Can we do a follow on PR for this item? I just finally figured out how to squash my commits and don't want to add to them now.

Reply:
Hi @cmastr. I am not sure what happened when you "squashed" your commits.
It looks like you have 6 commits for this pull request?
At the beginning of the PR you can see the commits listed and the files changed.
To move forward, you could try and squash again or open a new PR with your updated changes?